### PR TITLE
F 285 add sort by trace total donation

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related #285  #309 
@CarlosQ96 
Now we can get sorted list by `traceTotalDonations`, the num field is `TraceDonations`

BTW I have added test cases for get projects query

### CODE REVIEW

**Review by CarlosQ96 — APPROVED** [review]
Ran all tests and they worked fine! I shall fix the remaining tests in another PR. Retaking on prisma now that I am full time. So many changes so I'll adapt what I did with all this.

File: test/testUtils.ts [hunk 1]
```diff
@@ -51,6 +54,74 @@ export const generateTestAccessToken = async (id: number): Promise<string> => {
   );
 };
 
+export interface CreateProjectData {
```
Comment (CarlosQ96) [inline test/testUtils.ts:57]: I was thinking we have a seed file called helpers.ts Maybe we can reuse that? or adapt it for test env?

Comment (CarlosQ96) [reply test/testUtils.ts:57]: Not required for this PR but there are a bunch of models there

Comment (mohammadranjbarz) [reply test/testUtils.ts:57]: great idea, I had forgotten about that, I think in the future we can use that, thanks for mentioning

**Pull request merged**